### PR TITLE
docs: Small devtools docs fixes

### PR DESCRIPTION
**Pull request by erickzanardo**
# Description

Replace this text.

## Checklist

- [x] I have followed the [Contributor Guide] when preparing my PR.
- [ ] I have updated/added tests for ALL new/updated/fixed functionality.
- [ ] I have updated/added relevant documentation in `docs` and added dartdoc comments with `///`.
- [ ] I have updated/added relevant examples in `examples` or `docs`.

## Breaking Change?

- [ ] Yes, this PR is a breaking change.
- [x] No, this PR is not a breaking change.

## Related Issues

[Contributor Guide]: https://github.com/flame-engine/flame/blob/main/CONTRIBUTING.md
[Conventional Commit]: https://conventionalcommits.org
[CHANGELOG]: https://github.com/flame-engine/flame/blob/main/CHANGELOG.md

### CODE REVIEW

**Review by spydon — COMMENTED** [review]
You don't need to use `run` to run scripts

File: packages/flame_devtools/README.md [hunk 1]
```diff
@@ -29,11 +29,11 @@ ask you if you want to add the Flame DevTools extension as a separate tab.
 
 ## Development
 
-To run it locally, make sure to run `melos devtools-build` to build the
+To run it locally, make sure to run `melos run devtools-build` to build the
```
Comment (erickzanardo) [inline packages/flame_devtools/README.md:32]:
```suggestion
To run it locally, make sure to run `melos devtools-build` to build the
```

File: packages/flame_devtools/README.md [hunk 2]
```diff
@@ -29,11 +29,11 @@ ask you if you want to add the Flame DevTools extension as a separate tab.
...
 extension so that it can be loaded in the browser (the build files are not
 committed to the repository).
 
-After you have done any changes, make sure to run `melos devtools-build` to
+After you have done any changes, make sure to run `melos run devtools-build` to
```
Comment (erickzanardo) [inline packages/flame_devtools/README.md:36]:
```suggestion
After you have done any changes, make sure to run `melos devtools-build` to
```

File: packages/flame_devtools/README.md [hunk 3]
```diff
@@ -42,7 +42,7 @@ from the devtools extension. Don't forget to add the new connector to the
 list of connectors in the `DevToolsService` class.
 
 If you want to run with the devtools extension with the simulated mode for
-faster development, you can use `melos devtools-simulate` to start the
+faster development, you can use `melos run devtools-simulator` to start the
```
Comment (erickzanardo) [inline packages/flame_devtools/README.md:45]:
```suggestion
faster development, you can use `melos devtools-simulator` to start the
```

**Review by spydon — APPROVED** [review]

**Comment by erickzanardo** [comment]
> You don't need to use `run` to run scripts

You are right, was just the `s/simulate/simulator/g` that was needed.

**Pull request merged**